src/sagas.js

```diff
@@ -802,6 +808,60 @@ function* removeVm (action) {
   yield stopProgress({ vmId: action.payload.vmId, name: 'remove', result })
 }
 
+function* fetchAllEvents (action) {
+  const { userId } = action.payload
+  const events = yield callExternalAction('events', Api.events, { payload: {} })
+
+  if (events.error) {
+    return
+  }
+
+  const internalEvents = events.event
+    ? events.event
+      .filter((event) =>
+        event.severity === 'error' &&
```

Comment:
Well, yeah it can get not only errors, but previously VM Portal stored only error messages, and issue was about disappearing notifications which in that case has severity `error`.